src/server/live-reload/live-reload-server.js

```diff
@@ -1,32 +1,58 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 
-var config = require('../config');
 var ncp = require('ncp');
 var path = require('path');
-var prepare = require('../prepare');
 var Q = require('q');
-var telemetry = require('../telemetry-helper');
 var Watcher = require('./watcher').Watcher;
 
-var socket;
-var watcher;
+/**
+ * @constructor
+ */
+function LiveReload(project, telemetry, forcePrepare) {
```

Comment:
I was thinking that maybe we can rename the file to `live-reload.js`, since this is not handling any server.